javalib/src/main/scala/java/util/Random.scala

```diff
@@ -16,7 +17,11 @@ class Random(seed_in: Long) extends AnyRef with java.io.Serializable {
 
   setSeed(seed_in)
 
-  def this() = this(Random.randomSeed())
+  def this() = {
+    this(0) // required but don't use
```

Comment:
Apparently the `hashCode` is not set until after the main constructor completes so the test case I added will not pass. The `hashCode` guarantees the seed is different even if `Random`s get created in the same millisecond.